ibis/backends/flink/registry.py

```diff
@@ -454,6 +467,8 @@ def _timestamp_from_ymdhms(
     # ibis.expr.operations.numeric
     ops.IsNan,
     ops.IsInf,
+    # ibis.expr.operations.reductions
+    ops.ArrayCollect,
```

Comment:
Why do you need to add this? It doesn't exist in the base registry to begin with, so it has no real effect.